tests/testthat/test-run.R

```diff
@@ -610,3 +610,25 @@ test_that("can enable logging at the packet level", {
                fixed = TRUE, all = FALSE)
   expect_match(res$output, "orderly3::orderly_artefact")
 })
+
+
+
+test_that("cope with failed run", {
+  path <- test_prepare_orderly_example("implicit")
+  prepend_lines(file.path(path, "src", "implicit", "orderly.R"),
+                'd <- read.csv("something.csv")')
+  err <- expect_error(
+    orderly_run("implicit", root = path),
+    "cannot open the connection",
+    class = "outpack_packet_run_error")
```

Comment:
Wow this made me realise the message when reading a file "cannot open file 'thing.csv': No such file or directory" is actually a warning instead of the error :fearful:. It's a bit of a shame we can't show that message when a user runs `orderly_run` here but perhaps it's hard to work around as that is specific to the error handling of read.csv. It is available in the `output` of the object

Reply:
yeah, this is really annoying - it should be an error that is visible as part of the other error. Definitely not something that we should handle specifically. However, we could look at how something like callr shows some context on the screen at the point where the error is thrown?

Reply:
Yeah that sounds like a good plan